traffic_portal/app/src/common/api/ParameterService.js

```diff
@@ -30,9 +30,9 @@ var ParameterService = function(Restangular, $http, $q, locationUtils, messageMo
     this.createParameter = function(parameter) {
         return Restangular.service('parameters').post(parameter)
             .then(
-            function() {
+            function(response) {
                 messageModel.setMessages([ { level: 'success', text: 'Parameter created' } ], true);
-                locationUtils.navigateToPath('/parameters');
```

Comment:
Couldn't this be accomplished by just changing this path to `'/parameters/' + String(parameter.id) + '/profiles'` ?

Reply:
maybe? i need to read up on the  issue. the PR doesn't say exactly what the change is trying to accomplish.

Reply:
@ocket8888 - not quite. the issue created by @rawlinp (#3453) doesn't want a simple redirect to `'/parameters/' + String(new_parameter.id) + '/profiles' ` after creating a parameter (although that would be an improvement imo) but instead, wants the parameter/profile assignment modal opened after creating a parameter.

Reply:
Yeah, @mitchell852 is right. The idea is that if you just created a parameter, you most likely want to then assign that parameter to a profile.
So it seems like there would be two steps:

1. After creating a parameter, instead of navigating back to the "parameters table" view, navigate to the "profiles linked to this parameter" view.
2. Automatically open the "assign parameter to profiles" modal.
When you're done assigning the parameter to profiles, you should end up on the "profiles linked to this parameter" view.

Orphaned parameters don't make much sense, so the easier we can make it to assign parameters to profiles right after creating them, the better.

Reply:
I do feel like this could be simplified a lot however (code-wise) if the requirement changed to this:

- On successful parameter creation, redirect TP to /parameters/new_parameter.id/profiles where the user is required (if they want) to click the link button to link profiles to their new parameter

![image](https://user-images.githubusercontent.com/251272/61382225-98cb3f00-a869-11e9-98db-50f8869f180e.png)

this would turn all these code changes into a one line change :)

Reply:
....and if the above is OK, then the same could be done on successful profile creation (with another one line change)....

- On successful profile creation, redirect TP to /profiles/new_profile.id/parameters where the user is required (if they want) to click the link button to link parameters to their new profile.

2 lines of code == 2 new timesaving behaviors :)

Reply:
Well when you put it that way, I can get on board with that. If it obfuscates the code too much to auto-open the assignment modal, I'd be +1 on that trade-off.